misc/releaser.py

```diff
@@ -30,27 +32,126 @@ class ReleaseError(Exception):
     pass
 
 
-Version = namedtuple("Version", "full,short,major,minor,patch,extra")
-Version.__eq__ = lambda self, other: self.full == other.full
-Version.__lt__ = lambda self, other: (self.major, self.minor, self.patch) < (
-    other.major,
-    other.minor,
-    other.patch,
-)
+class Version:
```

Comment:
Why not simply use `from distutils.version import LooseVersion`?

Reply:
because... I didn't remember where this class was ! (I was looking for `type(sys.version_info)`)

But now that Version is here and fine tuned, I don't think we should switch to `distutils.version.LooseVersion`